### PR TITLE
test(components): Check the Sagemaker component output rather than Controller

### PR DESCRIPTION
**Pull request by ananth102**
**Description of your changes:**
- `components/aws/sagemaker/tests/integration_tests/component_tests/test_v2_hosting.py` -  check in components rather than controller
- `components/aws/sagemaker/tests/integration_tests/component_tests/test_v2_train_component.py` - Same as previous

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: components/aws/sagemaker/tests/integration_tests/component_tests/test_v2_hosting.py [hunk 1]
```diff
@@ -64,7 +66,41 @@ def test_create_v2_endpoint(kfp_client, experiment_id, boto3_session, test_file_
             k8s_client, input_endpoint_name, "endpoints"
         )
 
-        endpoint_describe["status"]["endpointStatus"] == "InService"
+        outputs = {
```
Comment (surajkota) [inline components/aws/sagemaker/tests/integration_tests/component_tests/test_v2_hosting.py:69]: can you add similar check for model and endpoint config? just arn should be sufficient

**Comment by gkcalat** [comment]
@surajkota, can you take another look as it seems that new commits were pushed. Thanks!

**Review by surajkota — APPROVED** [review]
/approve
/lgtm

**Pull request merged**